jobs/jumpbox/templates/config/bashrc

```diff
@@ -18,7 +18,8 @@ fi
 #####################################################  env vars you get for free
 
 export GOPATH=${HOME}/go
-export PATH=/var/vcap/packages/jumpbox/bin:$PATH:/var/vcap/bosh/bin:/var/vcap/packages/golang/bin:${GOPATH}/bin
+export PATH=/var/vcap/bosh/bin:${GOPATH}/bin:${PATH}
+for p in /var/vcap/packages/*/{s,}bin; do case $p in (*toolbelt*) true ;; (*) export PATH="${p}:${PATH}" ;; esac done
```

Comment:
please do not do this in a single line.  Why is this being done?  Do we not know the explicit packages we want as part of the path?

Reply:
I simplified this. We are still using something similar for matching ruby against a wildcard so that future version bumps don't require changes here. The current ruby path is `/var/vcap/packages/ruby-3.0` and a wildcard `*` is not permitted within the PATH.